tests/bsim/bluetooth/audio/overlay-bt_ll_sw_split.conf

```diff
@@ -1,45 +1,119 @@
-# Controller Settings
-CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=255
+# Host and Controller common dependencies
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_PER_ADV_SYNC_MAX=2
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_MAX_CONN=3
+
+# Broadcast and Connected ISO
+CONFIG_BT_ISO_BROADCASTER=y
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_CENTRAL=y
+CONFIG_BT_ISO_PERIPHERAL=y
```

Comment:
Since this is an overlay, what's the purpose or overlaying these values when they are already provided by the "main" conf file? 

Shouldn't the overlay only contain values that are different?

Reply:
I have removed the commit that makes the nrf52 same as nrf53... will address that in a separate PR, may be having different Kconfig for nRF52 and nRF53 could be better in long run to discover bugs.